config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
+  get 'users/show'
+
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   # root 'posts'
-
   devise_for :users
   resources :posts
-
+  resources :users
```

Comment:
let's nest the resource like
http://guides.rubyonrails.org/routing.html#nested-resources
https://stackoverflow.com/questions/27712947/nested-resources-in-devise